packages/brisa-eslint-plugin/index.ts

```diff
@@ -0,0 +1,70 @@
+import { ESLintUtils, type TSESTree } from '@typescript-eslint/utils';
+
+export const rules = {
+  'no-server-event-prevent': ESLintUtils.RuleCreator(
+    (name) => `https://eslint.org/docs/rules/${name}`,
+  )<[], 'preventUsage'>({
+    name: 'no-server-event-prevent',
+    meta: {
+      type: 'problem',
+      docs: {
+        description:
+          'Disallow preventDefault() and stopPropagation() on server events',
+      },
+      schema: [],
+      messages: {
+        preventUsage: 'Avoid using {{ method }} on server events.',
+      },
+    },
+    defaultOptions: [],
+    create(context) {
+      const SERVER_EVENTS = new Set([
+        'load',
```

Comment:
This should apply to all element events: `click`, `submit`, `touchStart`... Instead of window events.